### PR TITLE
KAFKA-4744: Increased timeout for bounce test

### PR DESCRIPTION
**Pull request by enothereska**

### CODE REVIEW

**Comment by enothereska** [comment]
@guozhangwang @ijuma this should address KAFKA-4744 that fails occasionally.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM. I'll wait for one review from another person who is more familiar with the Streams system tests (if it's @guozhangwang, he can just merge it).

**Review by mjsax — COMMENTED** [review]
LGTM

**Pull request closed without merging**
